apps/astarte_realm_management/test/astarte_realm_management/new_engine_test.exs

```diff
@@ -146,4 +148,28 @@ defmodule Astarte.RealmManagement.EngineTestv2 do
       end
     end
   end
+
+  describe "Test trigger policy" do
+    @describetag :trigger_policy
+
+    property "is installed correctly", %{realm: realm} do
+      check all(policy <- Astarte.Core.Generators.Triggers.Policy.policy()) do
+        policy_json = Jason.encode!(policy)
+        :ok = Engine.install_trigger_policy(realm, policy_json)
+
+        assert {:ok, _source} = Engine.trigger_policy_source(realm, policy.name)
```

Comment:
Nit: this does not check that what was installed is actually the same policy passed to `Engine.install_trigger_policy`, only that there exists a policy with the same name